README.md

```diff
@@ -18,314 +16,247 @@ Visit the full docs [here](https://PrefectHQ.github.io/prefect-dask) to see addi
         <img src="https://img.shields.io/badge/discourse-browse_forum-red.svg?color=0052FF&labelColor=090422&logo=discourse" /></a>
 </p>
 
-## Welcome!
-
-Prefect integrations with the [Dask.distributed](http://distributed.dask.org/) library for distributed computing in Python.
-
-Provides a `DaskTaskRunner` that enables flows to run tasks requiring parallel or distributed execution using Dask.
-
-## Getting Started
-
-### Python setup
-
-Requires an installation of Python 3.7+.
+![DALL·E 2023-01-10 15 19 31](https://user-images.githubusercontent.com/15331990/211682578-3e341709-6509-4c95-a6af-3b1160fe2961.png)
 
-We recommend using a Python virtual environment manager such as pipenv, conda, or virtualenv.
-
-These tasks are designed to work with Prefect 2.0. For more information about how to use Prefect, please refer to the [Prefect documentation](https://orion-docs.prefect.io/).
-
-### Installation
-
-Install `prefect-dask` with `pip`:
-
-```bash
-pip install -U prefect-dask
-```
-
-## Running tasks on Dask
-
-The `DaskTaskRunner` is a parallel task runner that submits tasks to the [`dask.distributed`](http://distributed.dask.org/) scheduler. 
-
-By default, a temporary Dask cluster is created for the duration of the flow run.
-
-For example, this flow counts up to 10 in parallel (note that the output is not sequential).
-
-```python
-import time
-
-from prefect import flow, task
-from prefect_dask import DaskTaskRunner
-
-@task
-def shout(number):
-    time.sleep(0.5)
-    print(f"#{number}")
+Visit the full docs [here](https://PrefectHQ.github.io/prefect-dask) to see additional examples and the API reference.
 
-@flow(task_runner=DaskTaskRunner)
-def count_to(highest_number):
-    for number in range(highest_number):
-        shout.submit(number)
+The `prefect-dask` collection makes it easy to include distributed processing for your flows. Check out the examples below to get started!
 
-if __name__ == "__main__":
-    count_to(10)
+## Integrate with Prefect flows
 
-# outputs
-#3
-#7
-#2
-#6
-#4
-#0
-#1
-#5
-#8
-#9
-```
+Perhaps you're already working with Prefect flows. It downloads many images to train your machine learning model--the problem is that it runs slow because it's executed sequentially.
```

Comment:
```suggestion
Perhaps you're already working with Prefect flows. Say your flow downloads many images to train your machine learning model. Unfortunately, the it takes a long time to download your flows because your code is running sequentially.
```